web/src/components/indexers/IndexersPage.tsx

```
@@ -47,22 +54,33 @@ export function IndexersPage({ withContainer = true }: IndexersPageProps) {
+      }
       setIndexers(data || [])
     } catch (error) {
-      toast.error("Failed to load indexers")
+      if (signal?.aborted) {
+        return
+      }
+      toast.error(tr("indexersPage.toasts.loadFailed"))
       setIndexers([])
     } finally {
+      if (signal?.aborted) {
+        return
```

Biome (2.4.4)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)